bundles/paikkatietoikkuna/coordinateconversion/Tile.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @class Oskari.bundle.cooridinateconversion.Tile
```

Comment:
Does this bundle really need a custom tile implementation? If not you should just extend the defaultextension/defaulttile. Also a typo on the cooridinateconversion.